### ISSUE TITLE
Creating directory if they don't exist

### ISSUE THREAD

**Issue by corneadoug** [jrnewell/goog-webfont-dl#7]
I just tried this plugin, its pretty awesome, however when trying to add a path with a folder that didn't exist, I get this error:
```
Running "goog-webfont-dl:patuaOne" (goog-webfont-dl) task
Start download of Google WebFont sourcefiles...Warning: events.js:154
      throw er; // Unhandled 'error' event
      ^

Error: ENOENT: no such file or directory, open 'src/fonts/npmFonts/Patua-One.css'
    at Error (native) Use --force to continue.
```

In this case, the folder `npmFonts` didn't exist

**Comment by jrnewell** [comment]
Fixed in version ``0.4.0``, sorry for the slow fix.

**Comment by corneadoug** [comment]
@jrnewell Awesome thank you
